src/services/doppler-legacy-client.ts

```diff
@@ -259,6 +306,97 @@ function mapNavMainEntry(json: any): MainNavEntry {
   };
 }
 
+function compareByPlanType(left: Plan, right: Plan): number {
+  if (
+    (left.type === 'prepaid' && right.type === 'monthly-deliveries') ||
+    (left.type === 'prepaid' && right.type === 'subscribers') ||
+    (left.type === 'subscribers' && right.type === 'monthly-deliveries')
+  )
+    return -1;
+  if (
+    (left.type === 'monthly-deliveries' && right.type === 'prepaid') ||
+    (left.type === 'subscribers' && right.type === 'prepaid') ||
+    (left.type === 'monthly-deliveries' && right.type === 'subscribers')
+  )
+    return 1;
+  return 0 || compareByFee(left, right);
+}
+
+function compareByFee(left: Plan, right: Plan): number {
+  if (getPlanFee(left) < getPlanFee(right)) return -1;
+  if (getPlanFee(left) > getPlanFee(right)) return 1;
+  return 0;
```

Comment:
In general the ternary operator is better to express this kind of things:

```suggestion
  return (getPlanFee(left) < getPlanFee(right)) ? -1
    : (getPlanFee(left) > getPlanFee(right)) ? 1
    : 0;
```